groups/hsp-iit.yml

```
@@ -17,6 +17,10 @@ hsp-iit/everyone:
   - "damianomal"
   - "yuriyOn"
   - "AngeloRendiniello"
+  - "claudioverardo"
+  - "elandini84"
+  - "fedeceola"
+  - "yuriyOn"
```

```suggestion
  - "claudioverardo"
  - "fedeceola"
```
`yuriyOn` and `elandini84` were already in.